CHANGELOG.md

```diff
@@ -177,11 +177,11 @@
 
 ### Fixes
 
+* **Use password** to load PDF with all modes
```

Comment:
Move this to the Features section and update the version here and in __version__.py. If the current version is `0.16.19-dev2`, the updated version will be `0.16.19-dev3`

Reply:
@Coniferish 
Done